pkg/server/server.go

```diff
@@ -238,10 +239,18 @@ func (s *Server) getTarget(target string) *Target {
 
 // Wrapper function to simplify writting error message responses. This function
 // is only intended to be used with the service and nothing else.
-func writeErrorResponse(w http.ResponseWriter, format string, a ...any) error {
+func writeErrorResponse(w http.ResponseWriter, logServer bool, format string, a ...any) error {
 	errmsg := fmt.Sprintf(format, a...)
-	log.Error().Msg(errmsg)
-	http.Error(w, errmsg, http.StatusInternalServerError)
+	if logServer {
+
+		log.Error().Msg(errmsg)
```

Comment:
Even though `true` is being passed for `logServer` in calls to `writeErrorResponse` above, I'm not seeing the log messages, even with `--verbose` on the server-side...

Reply:
I think this might be because it failing on the client side before a request is made to the server.

Reply:
(Irrelevant with [2b9e3d6](https://github.com/OpenCHAMI/configurator/pull/24/commits/2b9e3d66d2c4b114c2aee2fafcad26095a056761))